### ISSUE TITLE
Cannot connect Tradfri Remote and Switch

### ISSUE THREAD

**Issue by 1rfsNet** [ioBroker/ioBroker.zigbee#886]
Unfortunately I am unable to connect IKEA Tradfri Remote and the On/Off Switch (which are both supported according to the list) since updating the adapter to v.1.4.0 and switching to the CC26x2R1 board. I used both devices with my CC2531 stick in the past.

The interview with the device starts but then nothing happens:
2020-12-08 18:13:32.073 - info: zigbee.0 (9523) Zigbee: allowing new devices to join.
2020-12-08 18:14:05.498 - info: zigbee.0 (9523) Starting interview of '0x60a423fffef09770'
2020-12-08 18:14:33.335 - info: zigbee.0 (9523) Zigbee: stop joining
2020-12-08 18:16:33.665 - error: zigbee.0 (9523) Failed to interview '0x60a423fffef09770', device has not succesfully been paired. undefined

I already tried several times. Furthermore I tried to pair another device from Osram which worked totally fine on the first try.
Coordinator firmware version: {"type":"zStack3x0","meta":{"transportrev":2,"product":1,"majorrel":2,"minorrel":7,"maintrel":1,"revision":20201026}}

P.S. My Tradfri 30W Driver is working fine.

**Comment by 1rfsNet** [comment]
After restarting the adapter I get the following errors in log:

2020-12-09 17:10:05.197 - info: zigbee.0 (2442) 0x04cf8cdf3c7d2fb1 (addr 29632): Not supported (model undefined)(Unknown)
--
2020-12-09 17:10:05.197 - info: zigbee.0 (2442) 0x60a423fffef09770 (addr 64407): Not supported (model undefined)(Unknown)
2020-12-09 17:10:05.198 - info: zigbee.0 (2442) 0x680ae2fffea21721 (addr 13536): Not supported (model undefined)(Unknown)
2020-12-09 17:10:05.198 - info: zigbee.0 (2442) Zigbee started
2020-12-09 17:10:05.455 - error: zigbee.0 (2442) Device 0x04cf8cdf3c7d2fb1 "undefined" not described in statesMapping.
2020-12-09 17:10:05.458 - error: zigbee.0 (2442) Device 0x680ae2fffea21721 "undefined" not described in statesMapping.
2020-12-09 17:10:05.524 - error: zigbee.0 (2442) Device 0x60a423fffef09770 "undefined" not described in statesMapping.

**Comment by asgothian** [comment]
Please remove the non-working device and try to pair again. Please keep pressing a button on the remote every few seconds to ensure that it is awake and communicating.

Also please check the battery of the remote to ensure that it is not weak.

A

**Comment by 1rfsNet** [comment]
Already tried this several times, unfortunately no luck. Batteries are completely new but of course I also tried another one.

**Comment by asgothian** [comment]
Strange - can you please post the exact ikea model number of your device ? I am not aware of any ikea on/off switches not working with the cc26R1 board.

A.

**Comment by 1rfsNet** [comment]
Where do I find the model numbers?
It is this one: https://www.ikea.com/de/de/p/tradfri-kabelloser-dimmer-weiss-70408595/
NLE: 21633
2033 (just a number on the package, not sure what kind of number)

and this one: https://www.ikea.com/de/de/p/tradfri-fernbedienung-30443124/
NLE: 21533
2031

I have tried 2 of each of the devices, with all four I have the same problem. I therefore exclude a defect in the IKEA devices. I'm afraid I have no more ideas. The interview always starts but that's it. Always fails.

**Comment by asgothian** [comment]
There should be model descriptions in the format Exxxx printed on the devices. Those are the model numbers needed.

They are usually on the back of the device.

A.

**Comment by 1rfsNet** [comment]
I was able to fix it, you might want to fix it or note it somewhere:
I used the same instance I had already used for my CC2531 for the new board as well. I changed the Pan-ID there and re-connected all devices, which worked fine with all other devices, but with the IKEA devices it was probably a problem. I have now deleted the instance, created a new one and it works immediately.
The seller of the new board (which I asked for help) was able to reproduce this issue and was able to solve the problem for me.
